tests/async/test_evaluate.py

```diff
@@ -69,6 +69,14 @@ async def test_evaluate_transfer_bigint(page: Page) -> None:
     assert await page.evaluate("a => a", 17) == 17
 
 
+async def test_should_transfer_maps(page):
+    assert await page.evaluate("() => new Map([[1, { test: 42n }]])") == {}
```

Comment:
@mxschmitt Hi, does this mean playwright-python doesn't support transfering Map or Set actually?

```
Map(1) { 1 => { test: 42n } }
Set(2) { 1, { test: 42n } }
```

Reply:
Hi yes, we decided to not implement Map/Set in the language ports for now. (Then last 2 years nobody asked for it)

Reply:
Thank you for sharing about the decision :)